src/assisted_test_infra/test_infra/helper_classes/day2_cluster.py

```diff
@@ -88,9 +87,6 @@ def start_install_and_wait_for_installed(self):
         tfvars = utils.get_tfvars(self.config.tf_folder)
         tf_network_name = tfvars["libvirt_network_name"]
 
-        config = TerraformConfig()
-        config.nodes_count = num_nodes_to_wait
```

Comment:
What about this

Reply:
It's not doing anything (deprecated)
https://github.com/openshift/assisted-test-infra/blob/master/src/assisted_test_infra/test_infra/helper_classes/config/controller_config.py#L48-#L54